src/RoslynAnalyzers/Microsoft.CodeAnalysis.Analyzers/Microsoft.CodeAnalysis.Analyzers.sarif

```diff
@@ -5,7 +5,7 @@
     {
       "tool": {
         "name": "Microsoft.CodeAnalysis.Analyzers",
-        "version": "5.0.0",
+        "version": "5.1.0",
```

Comment:
@JoeRobich is this something we will have to do on every minor version bump?  Is it something we could extract from the roslyn version props?